### PR TITLE
Show opponents hand and deck size 

### PR DESCRIPTION
**Pull request by Ridleh**
Needs multiplayer testing. Deck and hand size will not update in AI battles because AI does not fire any property changes

### CODE REVIEW

File: Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java [hunk 1]
```diff
@@ -220,13 +222,34 @@ private void updateMP(int maxMP) {
...
         player2MPText.setFill(Color.AQUA);
         mpGroup.getChildren().add(player2MPText);
+        handleOpponentDeckView(maxMP);
+
+    }
+
+    public void handleOpponentDeckView(int maxMP){
+        Text opponentDeckInfo = new Text("");
+        if(battleScene.getMyPlayerNumber() == player1.getPlayerNumber()){
+            opponentDeckInfo.setText("Deck: " + player2.getDeckSize() + " Hand: " + player2.getHand().size() + "/6");
```
Comment (fluffy-hamster) [inline Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: "/6"  shouldn't be hard-coded.  use MAX_HAND_SIZE constant

File: Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java [hunk 2]
```diff
@@ -220,13 +222,34 @@ private void updateMP(int maxMP) {
                 e.printStackTrace();
             }
         }
+
         Text player2MPText = new Text(player2.getCurrentMP() + "/" + maxMP);
         player2MPText.setX(SCREEN_WIDTH - SCALE * (335 + maxMP * 40));
         player2MPText.setY(SCALE * (180 - maxMP * 2));
```
Comment (Jim9137) [inline Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: These` SCALE * (180 - maxMP * 2) `and `SCALE * (300 + maxMP * 40)` seem to pop up in several places. Would it make sense to introduce variable for these things, to reduce duplication?

File: Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java (hunk 1 above)
Comment (Ridleh) [reply Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: fixed

File: Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java (hunk 2 above)
Comment (Ridleh) [reply Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: Maybe? I think we'd need to simplify the math first to make it easier

Comment (Jim9137) [reply Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: I think first step could be just introduce constant for these equations. 'cause I have no idea why they are like. 😅

Comment (Ridleh) [reply Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: all of the capitalized variables are constants already ://///

Comment (Jim9137) [reply Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: Okay yeah, I took a bit clloser look, and it's not that easy. Whoops.  There are still some constants that could be extracted, but no biggie for now. 👍

Comment (Ridleh) [reply Client/src/main/java/org/projectcardboard/client/view/battleview/PlayerBox.java]: welcome to our spaghetti filled world :')

**Pull request merged**